Update image and add description for "Allow remote shopping assistance" and "Show Password" (#1468)
magento/merchdocs: src/customers/account-create.md

    This option appears even if the store doesn’t publish a newsletter.

> 1. If you want to allows merchants to [See what you see]({% link customers/login-as-customer.md %}) and take actions on your behalf in order to provide better assistance, select the **Allow remote shopping assistance** checkbox.

Reviewer: ```suggestion
1. If you want to enable store support staff to [see what you see]({% link customers/login-as-customer.md %}) and take actions on your behalf in order to provide better assistance, select the **Allow remote shopping assistance** checkbox.
```

    Then, enter it again to **Confirm Password**.

> 1. Click <span class="btn">Show Password</span> to see what password you are entering.

Reviewer: ```suggestion
1. If needed, click <span class="btn">Show Password</span> to see view the password you entered.
```
